packages/mobile/src/services/http/http.service.ts

```diff
@@ -83,6 +120,24 @@ class Http {
     return response;
   }
 
+  private async updateAuthorizationToken() {
+    const refreshToken = storage.getItem(StorageKey.REFRESH_TOKEN);
+
+    if (refreshToken) {
+      const payload = {
+        tokenValue: refreshToken,
+      };
+      const refreshResponse = await authApi.refreshToken(payload);
+
+      storage.setItem(StorageKey.ACCESS_TOKEN, refreshResponse.accessToken);
+      storage.setItem(StorageKey.REFRESH_TOKEN, refreshResponse.refreshToken);
+
+      if (!refreshResponse.accessToken) {
+        store.dispatch(authActions.logOut());
+      }
```

Comment:
It should go before storage set and I'd recommend throwing a custom error here and then handle it within error handler middleware